route/router.go

```diff
@@ -10,9 +10,35 @@ import (
 type Router interface {
 	// Options configures the router with the given options. An error is
 	// returned if validation issues exist.
-	Options(opts ...Option) error
+	Options(...Option) error
 	// Solver receives solve options and returns a Solver interface.
-	Solver(opt store.Options) (store.Solver, error)
+	Solver(store.Options) (store.Solver, error)
+	// Plan returns a variable which holds information
+	// about the current set of unassigned stops and vehicles.
+	// The plan variable can be used to retrieve the values from the
+	// store of a solution.
```

Comment:
```suggestion
	/*
		Plan returns a variable which holds information about the current set of
		vehicles with their respective routes and any unassigned stops. The Plan
		variable can be used to retrieve the values from the Store of a
		Solution.

			router, err := route.NewRouter(
				stops,
				vehicles,
				route.Capacity(quantities, capacities),
			)
			if err != nil {
				panic(err)
			}
			solver, err := router.Solver(store.DefaultOptions())
			if err != nil {
				panic(err)
			}
			solution := solver.Last(context.Background())
			s := solution.Store
			p := router.Plan()
			vehicles, unassigned := p.Get(s).Vehicles, p.Get(s).Unassigned
	*/
```

1. I changed the wording just a tiny bit to make it clear that you get the routes (talking about the vehicle routes before the unassigned stops).
2. I included the example you provided in the PR description here so that people have it handy when using intellisense.
3. Changed capitalization. I think throughout the SDK godocs, proper exported types are capitalized.

Reply:
Also feel free to dismiss if you don't agree.